src/cmd/bundle.js

```diff
@@ -35,17 +35,52 @@ module.exports = class BundleCommands extends CLICommandBase {
 			throw new Error(`The target file ${saveTo} must be a .zip file`);
 		}
 
-		// If no assets folder is specified, use the default assets folder in the current directory
-		let assetsPath = assets ? assets : 'assets';
-
+		let assetsPath = await this._getAssetsPath(assets);
 		const bundleFilename = this._getBundleSavePath(saveTo, appBinary);
 		return { assetsPath, bundleFilename };
 	}
 
+	async _getAssetsPath(assets) {
+		if (assets) {
+			if (await fs.exists(assets)) {
+				// check if assets is a project.properties file
+				const stat = await fs.stat(assets);
+				if (stat.isFile() && utilities.getFilenameExt(assets) === '.properties') {
+					return this._getAssetsPathFromProjectProperties(assets);
+				} else {
+					return assets;
+				}
+			}
+			throw new Error(`The assets folder ${assets} does not exist`);
+		}
+		const projectPropertiesPath = path.join(process.cwd(), 'project.properties');
+		return this._getAssetsPathFromProjectProperties(projectPropertiesPath);
+	}
+
+	async _getAssetsPathFromProjectProperties(projectPropertiesPath) {
+		if (!await fs.exists(projectPropertiesPath)) {
+			throw new Error('No project.properties file found in the current directory. ' +
+				'Please specify the assets directory using --assets option');
+		}
+		const propFile = await utilities.parsePropertyFile(projectPropertiesPath);
+		if (propFile.assetOtaFolder && propFile.assetOtaFolder !== '') {
+			// get the assets folder relative to the project.properties file
+			return path.join(path.dirname(projectPropertiesPath), propFile.assetOtaFolder);
+		} else if (propFile.assetOtaFolder === '') {
+			throw new Error('assetOtaFolder property in project.properties is empty.');
+		} else if (!propFile.assetOtaFolder) {
+			throw new Error('No assetOtaFolder property found in project.properties.');
```

Comment:
No need for 2 conditions. You can remove the empty string one and just keep the falsy condition. A call to action is better than an error message. `Add assetOtaFolder to your project.properties in order to bundle assets`

Make sure to also update the help for `particle bundle` to explain the new syntax: when `--asset` is missing it uses the path from `project.properties`. Add a note to the epilogue about the assetOtaFolder something like: `Add assetOtaFolder=assets to your project.properties to bundle the assets from the assets directory.`

Reply:
I asked chatgpt for some rewording suggestions for the rest